custodia/message/kem.py

```diff
@@ -373,7 +373,8 @@ def test_1_Parse_GET(self):
         kem = KEMHandler({'KEMKeysStore': self.kk})
         kem.parse(jtok, "mykey")
         out = kem.reply('output')
-        jtok = JWT(jwt=json_decode(out)['value'])
+        msg = json_decode(json_encode(out))
```

Comment:
no tone, I was just wondering.
I am fine with one test (with a comment that says why you are doing so), but I do not see thee need to repeat it identical in multiple places, as down the road people may get confused and either "clean it away" or just try to keep it for fear of breakking something even when not necessary.